docs/configuration/listeners/lds.rst

```diff
@@ -0,0 +1,77 @@
+.. _config_listeners_lds:
+
+Listener discovery service
+==========================
+
+The listener discovery service (LDS) is an optional API that Envoy will call to dynamically fetch
+listeners. Envoy will reconcile the API response and add, modify, or remove known listeners
+depending on what is required.
+
+The semantics of listener updates are as follows:
+
+* Every listener must have a unique :ref:`name <config_listeners_name>`. If a name is not
+  provided, Envoy will create a UUID. Listeners that are to be dynamically updated should have a
+  unique name supplied by the management server.
+* When a listener is added, it will be "warmed" before taking traffic. For example, if the listener
+  references an :ref:`RDS <config_http_conn_man_rds>` configuration, that configuration will be
+  resolved and fetched before the listener is moved to "active."
```

Comment:
Great, that's cleared it up for me mostly. So, the above only takes place during the init phase of the server at startup? On later LDS updates, we're never going to be waiting for the corresponding RDS update, right?